hedera-node/configuration/mainnet/application.properties

```diff
@@ -1 +1,12 @@
-
+autoRenew.targetTypes=
+contracts.knownBlockHash=055fac8eb788872957dfee45b7d40f7c94d66ea0a75ec703b993d57e441de574@33476932
+contracts.localCall.estRetBytes=4096
+hedera.allowances.isEnabled=true
+scheduling.whitelist=CryptoTransfer,ConsensusSubmitMessage,TokenBurn,TokenMint,CryptoApproveAllowance
+sigs.expandFromLastSignedState=false
+staking.fees.stakingRewardPercentage=10
+staking.maxDailyStakeRewardThPerH=6_849
+staking.maxStakeRewarded=650000000000000000
+staking.rewardBalanceThreshold=8500000000000000
+staking.rewardRate=48630136986000
+tokens.nfts.maxAllowedMints=10_000_000
```

Comment:
These are the values taken from mainnet as of Friday, September 15th which is running release `v0.40.x` code. Reviews should verify that these values are still applicable/correct for release `v0.42.x` series code.

Reply:
To my knowledge, these are correct